alerts/google-gke/uptime-checks-workload.v1.json

```diff
@@ -0,0 +1,37 @@
+{
+  "displayName": "${CLUSTER_NAME}/${WORKLOAD_NAME} GKE Load Balancer Check uptime failure"
+  "documentation": {},
+  "userLabels": {
+    "workload_name": "${WORKLOAD_NAME}",
+    "cluster": "${CLUSTER_NAME}",
+    "uptime_check_id": "${UPTIME_CHECK_ID}",
+  },
+  "conditions": [
+    {
+      "conditionThreshold": {
+        "aggregations": [
+          {
+            "alignmentPeriod": "1200s",
+            "perSeriesAligner": "ALIGN_NEXT_OLDER",
+            "crossSeriesReducer": "REDUCE_COUNT_FALSE",
+	    "groupByFields": [
+              "resource.label.*"
```

Comment:
Our current alert policy lists these out, but if we want to support ingress + load balancer through the same policy template then I think this is fine